pkg/cmd/release/create/create_test.go

```diff
@@ -1751,6 +1730,82 @@ func Test_createRun_interactive(t *testing.T) {
 	}
 }
 
+func Test_gitTagInfo(t *testing.T) {
+	const tagName = "foo"
+	const contentCmd = `git tag --list foo --format=%\(contents\)`
+	const signatureCmd = `git tag --list foo --format=%\(contents:signature\)`
+
+	tests := []struct {
+		name       string
+		runStubs   func(*run.CommandStubber)
+		wantErr    string
+		wantResult string
+	}{
+		{
+			name: "no signature",
+			runStubs: func(cs *run.CommandStubber) {
+				cs.Register(contentCmd, 0, "some\nmultiline\ncontent")
+				cs.Register(signatureCmd, 0, "")
+			},
+			wantResult: "some\nmultiline\ncontent",
+		},
+		{
+			name: "with signature",
+			runStubs: func(cs *run.CommandStubber) {
+				cs.Register(contentCmd, 0, "some\nmultiline\ncontent\n-----BEGIN SIGNATURE-----\nfoo\n-----END SIGNATURE-----")
+				cs.Register(signatureCmd, 0, "-----BEGIN SIGNATURE-----\nfoo\n-----END SIGNATURE-----")
+			},
+			wantResult: "some\nmultiline\ncontent",
+		},
+		{
+			name: "with signature in content but not as true signature",
+			runStubs: func(cs *run.CommandStubber) {
+				cs.Register(contentCmd, 0, "some\nmultiline\ncontent\n-----BEGIN SIGNATURE-----\nfoo\n-----END SIGNATURE-----")
+				cs.Register(signatureCmd, 0, "")
+			},
+			wantResult: "some\nmultiline\ncontent\n-----BEGIN SIGNATURE-----\nfoo\n-----END SIGNATURE-----",
+		},
```

Comment:
Is it important that we capture the 3 different signing signatures for testing purposes?

I know technically the code will take whatever is reported as the signature and remove it.

Reply:
You're right. I's best to keep tests unaware of the implementation details. I'll add proper test cases for different signatures.